Edits after Angela's comments (#4)
greenelab/nature_news_manuscript: content/02.introduction.md

However, we are interested in quantifying the additional level of bias beyond observed demographic differences in the scientific field, using authorship as an estimate for the existing demographic differences. 
This is similar to other studies that have quantified gender or racial disparities in science as observed in citation [@doi:10.1177/2378023117738903; @doi:10.1038/504211a] and funding rates [@doi:10.1016/j.cell.2021.01.011; @doi:10.1126/sciadv.aaz4868; @doi:10.1126/sciadv.aaw7238; @doi:10.1126/science.1165878; @doi:10.1126/science.1196783].
<!-- CUT and publication rates [@doi:10.1007/s11192-012-0893-4; @doi:10.1073/pnas.1915378117; @doi:10.1257/pandp.20181111]. -->
 
<!-- CUT It is crucial to ensure that science news coverage does not solely focus on a few well-known scientists, but expands our shared view of an expert scientist. -->
Even if science news coverage simply reflects the current scientific events and new findings, there are many gender, racial, or regional biases that can unknowingly seep into coverage.

> In researching a story, a journalist will typically interview multiple sources for their opinion, potentially asking for additional sources, thus allowing individual unconscious biases to skew scientific coverage broadly.

Reviewer: ```suggestion
In researching a story, a journalist will typically interview multiple sources for their opinion, potentially asking for additional sources, thus allowing individual unconscious biases at any point along the interview chain to skew scientific coverage broadly.
```
Not strong feeling at this, but I think this is what you are referring to more specifically. "at any point" might be removeable - not sure.

In addition, the repeated selection of a small set of field experts or the approach a journalist takes in establishing a new source may intensify existing biases [@https://www.theopennotebook.com/2016/08/23/including-diverse-voices-in-science-stories/; @https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
 

> While these biases may go unnoticed in a single article, analyzing a large corpus of articles can identify and quantify these biases and help guide institutional and individual self-reflection.

Reviewer: ```suggestion
While disparities in representation may go unnoticed in a single article, analyzing a large corpus of articles can identify and quantify these disparities and help guide institutional and individual self-reflection.
```

> In the same vein as previous media studies[@doi:10.1177/0003122415596999; @doi:10.1080/1461670X.2013.834149; @doi:10.1177/0163443711418272; @doi:10.1371/journal.pone.0148434; @https://www.poynter.org/reporting-editing/2013/lack-of-female-sources-in-new-york-times-stories-spotlights-need-for-change/; @https://whomakesthenews.org/gmmp-2015-reports/; @doi:10.1177/2378023117738903; @doi:10.1038/504211a; @doi:10.1007/s11192-012-0893-4; @doi:10.1073/pnas.1915378117; @doi:10.1257/pandp.20181111; @doi:10.1016/j.cell.2021.01.011; @doi:10.1126/sciadv.aaz4868; @doi:10.1126/sciadv.aaw7238; @doi:10.1126/science.1165878; @doi:10.1126/science.1196783], we sought to quantify gender and regional biases of news coverage beyond the existing demographic biases in the scientific field.

Reviewer: ```suggestion
In the same vein as previous media studies [@doi:10.1177/0003122415596999; @doi:10.1080/1461670X.2013.834149; @doi:10.1177/0163443711418272; @doi:10.1371/journal.pone.0148434; @https://www.poynter.org/reporting-editing/2013/lack-of-female-sources-in-new-york-times-stories-spotlights-need-for-change/; @https://whomakesthenews.org/gmmp-2015-reports/; @doi:10.1177/2378023117738903; @doi:10.1038/504211a; @doi:10.1007/s11192-012-0893-4; @doi:10.1073/pnas.1915378117; @doi:10.1257/pandp.20181111; @doi:10.1016/j.cell.2021.01.011; @doi:10.1126/sciadv.aaz4868; @doi:10.1126/sciadv.aaw7238; @doi:10.1126/science.1165878; @doi:10.1126/science.1196783], we sought to quantify gender and regional biases of news coverage beyond the existing demographic biases in the scientific field.
```
